app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  before_save :like_counter
```

Comment:
- [ ] As I mentioned above, it would be nice if you save it `after_save` you change rather than `before_save`. Also, do not forget to put your update method after `private`, ok.